src/antares/craft/model/study.py

```diff
@@ -219,8 +219,8 @@ def __init__(
         self._area_service = service_factory.create_area_service()
         self._link_service = service_factory.create_link_service()
         self._run_service = service_factory.create_run_service()
-        self._output_service = service_factory.create_output_service()
         self._binding_constraints_service = service_factory.create_binding_constraints_service()
+        self._output_service = service_factory.create_output_service()
```

Comment:
Don't think you need to create an output service here anymore

Reply:
The alternative would be to create the output service in local and api manually without the factory. Same thing could be said about binding_constraint_service which is not really needed in study either.
Or we could move read_outputs back into output_service but that would create redundancy with an output also having its own service.
I don't see what else I could do ?

Reply:
I think what you need to do is create the StudyService by giving it an output_service and with that you won't have to give the service inside the method

Reply:
I'll check

Reply:
I think what you should do is add a method set_output_service inside the study service that you'll call inside the create_study_service method. This way you can remove the service from the study as you'll have it inside the study_service. That's something we're doing already for the area_service if you want to check